src/components/Book.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledRow = styled.tr`
+  background: purple;
+  color: yellow;
+`;
+export default function Book(props) {
```

Comment:
In here; I think you can simplify by using arrow function like this 👇 right?
`const Book = ({ book: { title, category } }) => (
`
then make use of PropTypes.shape to capture all the props.